src/userLog.js

```diff
@@ -18,7 +18,7 @@ module.exports.write = function(user, network, chan, msg) {
 	var format = Helper.config.logs.format || "YYYY-MM-DD HH:mm:ss";
 	var tz = Helper.config.logs.timezone || "UTC+00:00";
 
-	var time = moment().utcOffset(tz).format(format);
+	var time = moment(msg.time).utcOffset(tz).format(format);
```

Comment:
Fixes `server-time`.

Reply:
Hmmm, does it? Won't this still use the weird utc offset (which defaults to UTC)?

Reply:
It fixes the fact it always used current time when writing log instead of the server time. So connecting to znc and receiving buffer would log 500 lines all having current time, instead of actual time when message was sent.

Reply:
Ah right, that was a different thing than what I thought you were saying it fixed. Cool, good catch.